### DISCUSSION TITLE
Flet FastAPI . canot show assets image

### DISCUSSION THREAD

**Discussion by bobwatcherx** [flet-dev/flet#2447] [Q&A]
### Question

i follow flet docs . but not work . you can see structure folder this
![Screenshot 2024-01-21 145317](https://github.com/flet-dev/flet/assets/52072811/61f604be-fca8-4fd8-9f38-d166c7b9a38a)
![Screenshot 2024-01-21 145308](https://github.com/flet-dev/flet/assets/52072811/58c30c41-5430-4e1d-94c8-30dcadfa3857)

### Code sample

```python
in picture
```

### Error message

```bash
not show image assets
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by AndrewRodriguesDev** [comment]
```python3
import flet as ft
import flet.fastapi as flet_fastapi
from fastapi.requests import Request
from fastapi.responses import RedirectResponse

# Configs:
ASSETS_DIR = '<assets-directory-absolute-path>'


# Views:
async def home(page: ft.Page):
    await page.add_async(
        ft.Text('Hello App!'), ft.Image(src='../../assets/<filename>')
    )


# Routes:
app = flet_fastapi.FastAPI()


@app.get('/')
async def _(request: Request):
    return RedirectResponse('/views/home')


app.mount('/views/home', flet_fastapi.app(home))


# Use this link to access files in assets directory: http://localhost:8000/assets/<filename>
app.mount('/assets', flet_fastapi.FletStaticFiles(assets_dir=ASSETS_DIR))

```
